doc/rst/usingchapel/chplenv.rst

```diff
@@ -417,10 +417,13 @@ CHPL_LOCALE_MODEL
         flat     top-level locales are not further subdivided
         numa     top-level locales are further subdivided into
                  sublocales, each one a NUMA domain
+        gpu      enable gpu sublocales
         ======== =============================================
 
    If unset, ``CHPL_LOCALE_MODEL`` defaults to ``flat``.
 
+   To enable GPU support, the value must be set to ``gpu``.
+
```

Comment:
Could please you add a `.. warning::` regarding the unstable nature of the GPU support and link to the technote for GPU support? You can draw inspiration from the warning at the top of https://chapel-lang.org/docs/technotes/gpu.html. A link would be just:
```
Please see :ref:`readme-gpu` for more information.
```